src/main/java/net/pms/database/MediaTableTVSeries.java

```diff
@@ -737,6 +782,18 @@ public static JsonObject getTvSeriesMetadataAsJsonObject(final Connection connec
 						result.addProperty("rating", MediaTableVideoMetadataIMDbRating.getValueForTvSerie(connection, id));
 						result.add("ratings", MediaTableVideoMetadataRatings.getJsonArrayForTvSerie(connection, id));
 						result.addProperty("released", MediaTableVideoMetadataReleased.getValueForTvSerie(connection, id));
+						if (lang != null && !"en-us".equalsIgnoreCase(lang)) {
+							VideoMetadataLocalized loc = MediaTableVideoMetadataLocalized.getVideoMetadataLocalized(connection, id, true, lang, imdbID, "tv", tmdbId, null, null);
+							if (loc != null) {
+								loc.localizeJsonObject(result);
+								//temp fix to store tmdbID if it was not before
```

Comment:
I'm hesitant about temporary fixes since they often stick around as we forget to remove them, and it can be hard to know when it's ok to remove them. If this is really the best strategy, can we leave a comment in the code that details exactly what the plan is, including when this can be removed?

Reply:
In fact, it is not a temp fix as the situation can be found even with the current code.

It test the presence of the tmdb ID, and store it if it was not present.
So it will happen only once.